runtime/moonbase/src/bridge_config.rs

```diff
@@ -0,0 +1,310 @@
+	pub TargetGlobalConsensusNetwork: NetworkId = NetworkId::ByGenesis(hex2array!("64d25a5d58d8d330b8804103e6452be6258ebfd7c4f4c1294835130e75628401"));
+	pub TargetGlobalConsensusNetworkLocation: Location = Location::new(
+		2,
+		[GlobalConsensus(TargetGlobalConsensusNetwork::get())]
+	);
+
+	pub TargetBridgeLocation: Location = Location::new(
+		2,
+		[
+			GlobalConsensus(TargetGlobalConsensusNetwork::get()),
+			TargetParachain::get()
+		]
+	);
+
+	pub UniversalAliases: BTreeSet<(Location, Junction)> = BTreeSet::from_iter(
+		alloc::vec![
+			(SelfLocation::get(), GlobalConsensus(TargetGlobalConsensusNetwork::get()))
+		]
+	);
+}
+
+impl Contains<(Location, Junction)> for UniversalAliases {
+	fn contains(alias: &(Location, Junction)) -> bool {
+		UniversalAliases::get().contains(alias)
+	}
+}
+
+/// Add GRANDPA bridge pallet to track the relay chain finality.
+pub type BridgeGrandpaInstance = pallet_bridge_grandpa::Instance1;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_westend::Westend;
+	type MaxFreeHeadersPerBlock = ConstU32<4>;
+	type FreeHeadersInterval = ConstU32<5>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = moonbase_weights::pallet_bridge_grandpa::WeightInfo<Runtime>;
+}
+
+pub type BridgeParachainsInstance = pallet_bridge_parachains::Instance1;
+impl pallet_bridge_parachains::Config<BridgeParachainsInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgesGrandpaPalletInstance = BridgeGrandpaInstance;
+	type ParasPalletName = ParasPalletName;
+	type ParaStoredHeaderDataBuilder = SingleParaStoredHeaderDataBuilder<BridgedChain>;
+	type HeadsToKeep = ParachainHeadsToKeep;
+	type MaxParaHeadDataSize = MaxParaHeadDataSize;
+	type WeightInfo = moonbase_weights::pallet_bridge_parachains::WeightInfo<Runtime>;
+}
+
+/// Add XCM messages support for Betanet->Stagenet
+pub type WithMessagesInstance = pallet_bridge_messages::Instance1;
+impl pallet_bridge_messages::Config<WithMessagesInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type ThisChain = ThisChain;
+	type BridgedChain = BridgedChain;
+	type BridgedHeaderChain =
+		pallet_bridge_parachains::ParachainHeaders<Runtime, BridgeParachainsInstance, BridgedChain>;
+
+	type OutboundPayload = XcmAsPlainPayload;
+	type InboundPayload = XcmAsPlainPayload;
+	type LaneId = bp_messages::HashedLaneId;
+
+	type DeliveryPayments = ();
+	type DeliveryConfirmationPayments = (); // Only necessary if we want to reward relayers
+
+	type MessageDispatch = BridgeXcmOver;
+	type OnMessagesDelivered = BridgeXcmOver;
+	type WeightInfo = moonbase_weights::pallet_bridge_messages::WeightInfo<Runtime>;
+}
+
+/// Add support for the export and dispatch of XCM programs withing
+/// `WithMessagesInstance`.
+pub type XcmOverInstance = pallet_xcm_bridge::Instance1;
```

Comment:
```suggestion
pub type XcmBridgeInstance1 = pallet_xcm_bridge::Instance1;
```

More generally, can you improve the naming to make it more self-descriptive of its purpose?